### ISSUE TITLE
Conflicting layout constraints

### ISSUE THREAD

**Issue by brendenwest** [ResearchKit/AppCore#111]
I'm getting layout constraint warnings like below (APCStudyOverviewCollectionViewController) for many AppCore view controllers out of the box. That is with no changes on my part. 

Has anyone else seen these?

---

2015-07-22 15:18:00.452 myApp[3046:271833] Unable to simultaneously satisfy constraints.
    Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSLayoutConstraint:0x7fa546014d30 V:[UIView:0x7fa546012cd0(343)]>",
    "<NSLayoutConstraint:0x7fa5460153e0 UIView:0x7fa546012cd0.top == UIView:0x7fa546012bc0.topMargin - 8>",
    "<NSLayoutConstraint:0x7fa546015480 UIView:0x7fa546012bc0.bottomMargin == UIView:0x7fa546012cd0.bottom - 8>",
    "<NSAutoresizingMaskLayoutConstraint:0x7fa546018090 h=--& v=--& V:[UIView:0x7fa546012bc0(50)]>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x7fa546014d30 V:[UIView:0x7fa546012cd0(343)]>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.

**Comment by p2** [comment]
Yes. Some were fixed, some re-introduced conflicting constraints.
